RELEASE.md

```diff
@@ -1,3 +1,9 @@
+# (unreleased)
+- added Model.remove_fixable_singularities to allow removing fixable singularities in the model's equations.
+
+
+  See for more details appendix B in: Johnstone, R. H. (2018). Uncertainty characterisation in action potential modelling for cardiac drug safety [PhD thesis]. University of Oxford. https://ora.ox.ac.uk/objects/uuid:0a28829c-828d-4641-bfb0-11193ef47195
```

Comment:
Can this link go in the docs somewhere? If it stays here it should be made clear which part of the changelog it refers to and the link should maybe be in a proper markdown format. The `[PhD thesis]` bit could cause problems too?